module/data/item/templates/activities.mjs

```diff
@@ -231,8 +234,8 @@ export default class ActivitiesTemplate extends SystemDataModel {
     if ( !source.system.actionType && !source.system.activation?.type
       && (source.type !== "tool") ) return false;
 
-    // If item was updated after `4.0.0`, it shouldn't need the migration
-    if ( !foundry.utils.isNewerVersion("4.0.0", source._stats?.systemVersion ?? "0.0.0") ) return false;
+    // If item was updated after `4.0.1`, it shouldn't need the migration
+    if ( !foundry.utils.isNewerVersion("4.0.1", source._stats?.systemVersion ?? "0.0.0") ) return false;
```

Comment:
We don't want to run the risk of re-migrating items that have already migrated to activities, but it feels reasonably safe to bump this version check in order to catch the items that were missed with the original migration. Any item that already has activities won't get hit a second time, and it's probably too early for many people to be doing things like deleting activities off their items that auto-migrated in 4.0.